Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -2741,9 +2741,6 @@ public void spawnPlayerAtRPC(float posX, float posY, float posZ, PhotonMessageIn
         
         private void Start()
         {
-            QualitySettings.vSyncCount = 1;
```

Comment:
Removing these lines of codes causing the game to run without a FPS limit when you are starting the game. This caused #351

Reply:
should be good to go now!

Reply:
I only got rid of the "QualitySettings.vSyncCount = 1;" reference as this will permanently lock the fps with VSync, Which resulted in the limiter not being able to be used at all. Setting the fps upon start in the manager is fine on it's own as this does not lock the FPS and block the FPSLimiter from working.